test/parallel/test-regress-GH-13435.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
```

Comment:
I copied the format of the existing regression tests. Any suggestion? Can only think of something like `test-http-socket-server-not-null` which seems as obscure as the current one.